src/spikeinterface/sorters/external/kilosortbase.py

```diff
@@ -121,18 +121,34 @@ def _get_specific_options(cls, ops, params):
     def _setup_recording(cls, recording, sorter_output_folder, params, verbose):
         cls._generate_channel_map_file(recording, sorter_output_folder)
 
-        if recording.binary_compatible_with(dtype="int16", time_axis=0, file_paths_lenght=1):
+        skip_kilosort_preprocessing = params.get("skip_kilosort_preprocessing", False)
+
+        if (
+            recording.binary_compatible_with(dtype="int16", time_axis=0, file_paths_lenght=1)
+            and not skip_kilosort_preprocessing
+        ):
```

Comment:
I think you are gona hate this : P

Reply:
of course.
the waste of energy combine with the indigestible results push me deeply toward a permanent grumpy state.
you should feel guilty man.